docs/articles/vs-test-adapter/NUnit-And-Microsoft-Test-Platform.md

```diff
@@ -38,13 +38,13 @@ You can switch between `VSTest` and `dotnet test` using a property in your cspro
  <TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>
 ```
 
-See [this article](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-integration-dotnet-test#dotnet-test---microsofttestingplatform-mode)
+See [this article](https://learn.microsoft.com/dotnet/core/testing/unit-testing-platform-integration-dotnet-test#dotnet-test---microsofttestingplatform-mode)
 for more information.
 
-## Information on the Microsoft Test Platform
+## Information on the Microsoft.Testing.Platform
 
-* [Microsoft Test Platform](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-intro?tabs=dotnetcli)
-* [Use Microsoft.Testing.Platform with dotnet test](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-integration-dotnet-test)
-* [Microsoft Test Platform Architecture](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-architecture)
+* [Microsoft.Testing.Platform](https://learn.microsoft.com/dotnet/core/testing/unit-testing-platform-intro)
+* [Use Microsoft.Testing.Platform with dotnet test](https://learn.microsoft.com/dotnet/core/testing/unit-testing-platform-integration-dotnet-test)
```

Comment:
🏆 Nice catch on removing the default `en-us`.

Reply:
Credit to @Evangelink :)